frame/support/test/tests/benchmark_ui/empty_function.stderr

```diff
@@ -0,0 +1,5 @@
+error: Benchmark function definitions cannot be empty!
```

Comment:
Huh, couldn't this also be handled by the "no valid `#[extrinsic_call]` or `#[block]`" error? Why the need to create a separate error message here?

Reply:
It was originally actually. I thought it would be more informative this way, and I also had a let/else here that needs to throw an error in the else: https://github.com/paritytech/substrate/blob/460ed3c9f1bcdfdbf11dfe1dd68b31b2d1163894/frame/support/procedural/src/benchmark.rs#L308

Reply:
Got it, all set with latest commit